lighthouse-core/audits/seo/robots-txt.js

```diff
@@ -108,44 +108,55 @@ function parseLine(line) {
   };
 }
 
+/**
+ * @param {string} content
+ * @returns {Array<{index: string, line: string, message: string}>}
+ */
 function validateRobots(content) {
+  /**
+   * @type Array<{index: string, line: string, message: string}>
+   */
+  const errors = [];
   let inGroup = false;
 
-  return content
+  content
     .split(/\r\n|\r|\n/)
-    .map((line, index) => {
+    .forEach((line, index) => {
       let parsedLine;
 
       try {
         parsedLine = parseLine(line);
       } catch (e) {
-        return {
-          index: index + 1,
+        errors.push({
+          index: (index + 1).toString(),
```

Comment:
`Audit.makeTableDetails` type definition forces all of these fields to be strings